scripts/bench_swe_lite_dry.py

```
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-# SWE-bench Lite (DRY) – agent latency/behavior; free-tier friendly
+# SWE-bench Lite (DRY) – agent latency/behaviour; free-tier friendly
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)